app/src/main/java/com/example/parkingsystemjava/mvp/model/ParkingModel.java

```diff
@@ -1,13 +1,16 @@
 package com.example.parkingsystemjava.mvp.model;
 
+import com.example.parkingsystemjava.database.ReservationDatabase;
 import com.example.parkingsystemjava.mvp.contract.ParkingContract;
 
 public class ParkingModel implements ParkingContract.MainActivityModel {
     private String parkingLotsAvailable;
+    private ReservationDatabase database;
 
     @Override
     public void setParkingLots(String parkingLotsAvailable) {
-        this.parkingLotsAvailable = parkingLotsAvailable;
+        database = ReservationDatabase.getInstance();
```

Comment:
should not instantiate the database here.
this couldnt be tested. think in Dependency Inversion.
This should be passed as a parameter in the constructor (injected)

Reply:
done